### DISCUSSION TITLE
Add regressors

### DISCUSSION THREAD

**Discussion by AlejandroGB13** [ourownstory/neural_prophet#591] [Q&A - get help using NeuralProphet] [answered]
Hey,
Can it be possible to add regressors in neural prophet as well as was in prophet?

I'm trying to run something like this:
m.add_regressor('addr1')

And getting this error:
'NeuralProphet' object has no attribute 'add_regressor'

Thank you.

**Comment by ourownstory** [comment, marked as answer]
NeuralProphet supports both future and lagged regressors.

The equivalent to the prophet function is `add_future_regressor`

https://github.com/ourownstory/neural_prophet/blob/7d0135a0c9b02b1a0c18b66cb2310391600de2fb/neuralprophet/forecaster.py#L402
